src/scripts/background.js

```diff
@@ -2254,22 +2253,6 @@ window.TogglButton = {
     }
   },
 
-  checkPermissions: async function (show) {
-    const dontShowPermissions = await db.get('dont-show-permissions');
```

Comment:
This PR is good to go!

There's some additional code we can get rid of what do you think?

![image](https://user-images.githubusercontent.com/1716853/74429295-b3f24180-4e80-11ea-8eb9-fde130532663.png)

Reply:
Pushed a commit 🙆‍♂